Functions.Templates/Templates/EventGridTrigger-PowerShell/run.ps1

```diff
@@ -0,0 +1,3 @@
+param($eventGridEvent, $TriggerMetadata)
+
+Write-Host $eventGridEvent

```

Comment:
`Write-Host $input` and `$input | Write-Host` are equivalent.  I don't recall if the language worker does something special with `Write-Host` passing it to `Out-String`?  If so, it should render fine as a hashtable, otherwise, you would still get the `ToString()` output which will be the typename.

Reply:
Ah yeah good catch. We don't pass it to `Out-String`. We DO pass the object through Out-String if it's written to the pipeline so the code should be just:

```suggestion
$eventGridEvent
```